README.md

```diff
@@ -37,6 +37,14 @@ In the `dependencies.gradle` file add:
 
 ```groovy
 dependencies {
-    runtimeOnly 'uk.ac.ox.softeng.maurodatamapper.plugins:mdm-plugin-database-postgresql:4.0.0'
+    runtimeOnly 'uk.ac.ox.softeng.maurodatamapper.plugins:mdm-plugin-database-postgresql:5.0.0'
```

Comment:
please change this back, i will make this update once i release

Reply:
done